specification/cognitiveservices/data-plane/Luis/v2.0/LUIS Endpoint API.swagger.json

```diff
@@ -0,0 +1,268 @@
+            "in": "query",
+            "description": "The query to predict",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "timezoneOffset",
+            "in": "query",
+            "description": "The timezone offset for the location of the request",
+            "type": "number"
+          },
+          {
+            "name": "verbose",
+            "in": "query",
+            "description": "If true will return all intents instead of just the topscoring intent",
+            "type": "boolean"
+          },
+          {
+            "name": "spellCheck",
+            "in": "query",
+            "description": "Enable spell checking",
+            "type": "boolean"
+          },
+          {
+            "name": "staging",
+            "in": "query",
+            "description": "Use staging endpoint",
+            "type": "boolean"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A JSON object containing the predictions",
+            "examples": {
+              "application/json": {
+                "query": "forward to frank 30 dollars through HSBC",
+                "topScoringIntent": {
+                  "intent": "give",
+                  "score": 0.3964121
+                },
+                "entities": [
+                  {
+                    "entity": "30",
+                    "type": "builtin.number",
+                    "startIndex": 17,
+                    "endIndex": 18,
+                    "resolution": {
+                      "value": "30"
+                    }
+                  },
+                  {
+                    "entity": "frank",
+                    "type": "frank",
+                    "startIndex": 11,
+                    "endIndex": 15,
+                    "score": 0.935219169
+                  },
+                  {
+                    "entity": "30 dollars",
+                    "type": "builtin.currency",
+                    "startIndex": 17,
+                    "endIndex": 26,
+                    "resolution": {
+                      "unit": "Dollar",
+                      "value": "30"
+                    }
+                  },
+                  {
+                    "entity": "hsbc",
+                    "type": "Bank",
+                    "startIndex": 36,
+                    "endIndex": 39,
+                    "resolution": {
+                      "values": [
+                        "BankeName"
+                      ]
+                    }
+                  }
+                ]
+              }
+            }
+          }
+        },
+        "produces": [
+          "application/json"
+        ]
+      },
+      "post": {
+        "description": "Gets the published endpoint prediction for the given long query",
+        "operationId": "v2.0 Response - Get predictions from endpoint (post)",
+        "parameters": [
+          {
+            "name": "appId",
```

Comment:
if parameters are shared among operations you could define them in global section and refer them from here. 
https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#parameters-definitions-object
Make sure to include https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-parameter-location as needed.